ext/standard/head.c

```diff
@@ -238,6 +243,7 @@ static void php_setcookie_common(INTERNAL_FUNCTION_PARAMETERS, bool is_raw)
 		Z_PARAM_STR(domain)
 		Z_PARAM_BOOL(secure)
 		Z_PARAM_BOOL(httponly)
+		Z_PARAM_BOOL(partitioned)
```

Comment:
Please do not add new parameters. `SameSite` is only supported within the `$options` array as well. 

see also: #10317.